content/telegraf/v1.15/administration/windows_service.md

```diff
@@ -8,41 +8,59 @@ menu:
-
-   ```
-   > C:\"Program Files"\Telegraf\telegraf.exe --service install
-   ```
+| Command                            | Effect                        |
+|------------------------------------|-------------------------------|
+| `telegraf.exe --service install`   | Install telegraf as a service |
+| `telegraf.exe --service uninstall` | Remove the telegraf service   |
+| `telegraf.exe --service start`     | Start the telegraf service    |
+| `telegraf.exe --service stop`      | Stop the telegraf service     |
 
-5. Edit the configuration file to meet your requirements.
+Outlined below are the general steps to install Telegraf as a Service.
 
-6. To verify that it works, run:
+{{% note %}}
+Installing a Windows service requires administrative permissions.
+Be sure to [launch Powershell as administrator](
+https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
+{{% /note %}}
 
+1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
+2. In PowerShell, run the following as an administrator:
+   ```powershell
+   > cd "C:\Program Files\InfluxData\Telegraf"
+   > .\telegraf.exe --service install --config "C:\Program Files\InfluxData\Telegraf\telegraf.conf"
    ```
+   When installing as service in Windows, always double check to specify full, correct path of the config file.
+   Otherwise the Windows service may fail to start.
+3. To test that the installation works, run:
+
+   ```powershell
    > C:\"Program Files"\Telegraf\telegraf.exe --config C:\"Program Files"\Telegraf\telegraf.conf --test
    ```
 
-7. To start collecting data, run:
+4. To start collecting data, run:
 
-   ```
-   > net start telegraf
+   ```powershell
+   telegraf.exe --service start
    ```
 
-## Other supported operations
+<!--
+#### Config Directory
 
-Telegraf can manage its own service through the `--service` flag:
+You can also specify a `--config-directory` for the service to use:
 
-| Command                            | Effect                        |
-|------------------------------------|-------------------------------|
-| `telegraf.exe --service install`   | Install telegraf as a service |
-| `telegraf.exe --service uninstall` | Remove the telegraf service   |
-| `telegraf.exe --service start`     | Start the telegraf service    |
-| `telegraf.exe --service stop`      | Stop the telegraf service     |
+1. Create a directory for configuration snippets: `C:\Program Files\Telegraf\telegraf.d`
+2. Include the `--config-directory` option when registering the service:
+   ```
+   > C:\"Program Files"\Telegraf\telegraf.exe --service install --config C:\"Program Files"\Telegraf\telegraf.conf --config-directory C:\"Program Files"\Telegraf\telegraf.d
+   ```
+-->
```

Comment:
Is there a reason this is commented out? I support coming for this later?

Reply:
I had not been able to adequately test this yet in Windows, so leaving it commented for now.